deeds-dapp-webapp/src/main/webapp/vue-app/dapp/layout/components/Navbar.vue

```diff
@@ -389,6 +389,12 @@ export default {
         if (this.isMobile) {
           this.handlePageOpenedMobile();
         }
+        fetch(link, {
+          method: 'GET',
+          credentials: 'include',
+        })
+          .then(resp => resp?.ok && resp.text())
+          .then(text => window.document.head.innerHTML = text);
```

Comment:
To add inside `selectLanguage` method in main-*.js files + we will not add the whole content in existing `head`.

Reply:
done!